sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -2312,6 +2337,10 @@ public PCollection<KV<ByteString, ChangeStreamMutation>> expand(PBegin input) {
       if (getCreateOrUpdateMetadataTable() != null) {
         shouldCreateOrUpdateMetadataTable = getCreateOrUpdateMetadataTable();
       }
+      Duration backlogReplicationAdjustment = getBacklogReplicationAdjustment();
```

Comment:
Nit: if using Optional return type can just be 

'= getBacklogReplicationAdjustment().orElse(DEFAULT_BACKLOG_REPLICATION_ADJUSTMENT);

Reply:
nullable is fine. For streaming IO a consideration is upgrade compatibility, and newly added parameters then need to be nullable and set default in pipeline submission